src/components/Modal/FeedbackModal/styled.ts

```diff
@@ -5,14 +5,13 @@ export const Content = styled.div`
   background-color: white;
   padding: 16px 20px;
   border-radius: 8px;
-  width: 93%;
+  width: 80%;
```

Comment:
```suggestion
  width: 100%;
  max-width:375px;
```

이거는 제가 처음에 PC 화면 고려 안하고 잘못 만들었네요 이런식으로 하면 될 것 같아요